src/components/Proposals/Proposals.js

```diff
@@ -1,15 +1,36 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
+import { CardLayout, GU, useLayout } from '@aragon/ui'
+import { createAppHook, useApp } from '@aragon/connect-react'
+import connectVoting from '@aragon/connect-voting-disputable'
+import ProposalCard from './ProposalCard'
+
+const useDisputableVoting = createAppHook(connectVoting)
 
 function Proposals() {
+  const [disputableVoting] = useApp('disputable-voting')
+  const [votes] = useDisputableVoting(disputableVoting, (app) => app.votes())
```

Comment:
I think this works for now, but for organisation we will likely want to separate this into a hook `useDisputableVotes` or something :)

Reply:
yes! I also added it to app-logic (to have it all there)